chapter2_mdw/ExerciseThree

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
+
+use 5.010;
+use warnings;
+
+chomp($radius = <STDIN>);
+$pi = 3.141592654;
+
+$circumference = 2*$pi*$radius;
+
+if ($radius <= 0){
+  say "0"
```

Comment:
Could have set $circumference to 0 here, then had one say statement outside of the if construct. Maybe neater, maybe not?